docs/cmdline-opts/get.md

```diff
@@ -21,8 +21,11 @@ Example:
 
 When used, this option makes all data specified with --data, --data-binary
 or --data-urlencode to be used in an HTTP GET request instead of the POST
-request that otherwise would be used. The data is appended to the URL
-with a '?' separator.
+request that otherwise would be used. curl will append provided data to URL
+as a query string.
+
+Prior to 7.86.0 curl could use '?' as a separator between data fields in some
+edge cases.
```

Comment:
This seems to refer to a bug that we have not been notified about which also is already fixed.

If we should document an old bug in the current manpage, I think we need more details and a more specific description. But I will repeat that I don't think that this bug seems important enough to warrant a mention in current documentation.

I don't think this suggested wording will help any users.

Reply:
The only scenario to reproduce this problem is to:
- have data with forward slash (`/`) in key or value provided already in URL
- add additional data using `--data` or `--data-urlencode` (this may be valid to all others variants of --data-XXX - I was not testing)
- have curl older than 7.86.0 😉

As in example provided here: https://github.com/curl/curl/pull/13706#discussion_r1606396390

Not to make manual page too overwhelming, I called it 'edge case'. I hope if anyone notices `?` used as separator between data fields (instead only between URL and query string) and finds this description in manual page, he will be able to figure out that he is using too old version of curl. So he will upgrade instead of reporting issue, etc.

It's hard to match this behavior with correction description in changelog as it is called:
> [single_transfer: use the libcurl URL parser when appending query parts](https://curl.se/bug/?i=9681)

So maybe it somehow helps to have it mentioned additionally like this.

It was done a bit like here:
- https://github.com/curl/curl/blob/master/docs/cmdline-opts/disable.md
- https://github.com/curl/curl/blob/master/docs/cmdline-opts/include.md

Reply:
This bug happened if there was an existing *query* in the provided URL and that query contained a slash. Then curl would wrongly append the provided `--data` to the right side of the query with a `?` separator because the slash to the right side of the question mark makes it think there is no query part.

I still don't think this is a bug important enough to mention in the manpage, 17 releases after it was fixed.

Reply:
OK, part related to edge case is removed. I kept the other portion of the change. If not accepted let's close this PR.